src/main/java/ch/njol/skript/entity/SimpleEntityData.java

```diff
@@ -296,7 +296,8 @@ public boolean equals(final @Nullable Object obj) {
 		types.add(new SimpleEntityDataInfo("human", HumanEntity.class, true));
 		types.add(new SimpleEntityDataInfo("damageable", Damageable.class, true));
 		types.add(new SimpleEntityDataInfo("monster", Monster.class, true)); //I don't know why Njol never included that. I did now ^^
-		types.add(new SimpleEntityDataInfo("mob", Mob.class, true)); //Same goes for this one
+		if (Skript.classExists("org.bukkit.entity.Mob")) // Apparently not in 1.9.4
```

Comment:
It is added in 1.13.x, not 1.10. Do you use the Spigot docs you are hosting?

Anyways this comment is unnecessary
```suggestion
		if (Skript.classExists("org.bukkit.entity.Mob"))
```

Reply:
Ahh you are right, my bad!